sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -39,6 +41,11 @@
 @Experimental
 public interface SupportsNamespaces extends CatalogPlugin {
```

Comment:
hmm, is it better to name it `NamesapceCatalog`?

Reply:
Yea, make sense.